recipes/sqlite3/all/conanfile.py

```diff
@@ -94,17 +96,14 @@ def build(self):
     def package(self):
         header = tools.load(os.path.join(self._source_subfolder, "sqlite3.h"))
         license_content = header[3:header.find("***", 1)]
-        tools.save("LICENSE", license_content)
-
-        self.copy("LICENSE", dst="licenses")
-
+        tools.save(os.path.join(self.package_folder, "licenses", "LICENSE"), license_content)
         cmake = self._configure_cmake()
         cmake.install()
 
     def package_info(self):
         self.cpp_info.libs = tools.collect_libs(self)
         if self.settings.os == "Linux":
             if self.options.threadsafe:
-                self.cpp_info.libs.append("pthread")
-            if self.options.omit_load_extension == "False":
-                self.cpp_info.libs.append("dl")
+                self.cpp_info.system_libs.append("pthread")
+            if not self.options.omit_load_extension:
+                self.cpp_info.system_libs.append("dl")
```

Comment:
```suggestion
                self.cpp_info.system_libs.append("dl")
            if self.options.build_executable:
                bin_path = os.path.join(self.package_folder, "bin")
                self.output.info("Appending PATH env var with : {}".format(bin_path))
                self.env_info.PATH.append(bin_path)
```

Reply:
Thanks ! (realized the issue just after submitting PR and forgot to push the fix afterwards)